src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -822,22 +822,29 @@ export function Root(props: IProps): React.ReactElement {
                 }}
               >
                 {openScripts.map(({ fileName, hostname }, index) => {
+                  const editingCurrentScript = currentScript?.fileName === openScripts[index].fileName &&
+                    currentScript?.hostname === openScripts[index].hostname
+                  const externalScript = hostname !== 'home'
+                  const colorProps = editingCurrentScript ? {
+                    background: Settings.theme.button,
+                    borderColor: Settings.theme.button,
+                    color: Settings.theme.primary
+                  } : {
+                    background: Settings.theme.backgroundsecondary,
+                    borderColor: Settings.theme.backgroundsecondary,
+                    color: Settings.theme.secondary
+                  }
+
+                  if (externalScript) {
+                    colorProps.color = Settings.theme.error
```

Comment:
```suggestion
                    colorProps.color = Settings.theme.info
```

Using `Settings.theme.error` feels a bit out of place here. It makes it seem like something's wrong with the tab, rather than just indicating a difference of origin.

I think that changing it to `Settings.theme.info` or something else less "alarming" would help it still draw attention to the difference without giving the wrong idea :)